eventhubs-streamanalytics-eventhubs/tools/eh-asa-perfmon/EventHandler.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Azure.Messaging.EventHubs.Consumer;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace StreamingAtScale
+{
+    public class EventReceiver
+    {
+        private readonly object _lock = new object();
+
+        public async Task Receive(string connectionString, CancellationToken _cancellationToken)
+        {
+            await using (var consumerClient = new EventHubConsumerClient(EventHubConsumerClient.DefaultConsumerGroupName, connectionString))
+            {
+                Console.WriteLine("The application will now start to listen for incoming message.");
+
+                ReadEventOptions readOptions = new ReadEventOptions
```

Comment:
This is a really small wait time and definitely not something that I'd recommend for real-world use.  We should add some comments here to that effect and consider making it at least 500ms if not 1 second.

Reply:
resovled